examples/kafka/kafka.json

```diff
@@ -0,0 +1,26 @@
+{
+  "jobName": "kafka",
+  "image": "openpai/pai.example.kafka",
+  "taskRoles": [
+    {
+      "name": "main",
+      "taskNumber": 1,
+      "cpuNumber": 1,
+      "memoryMB": 2048,
+      "gpuNumber": 0,
+      "command": "/bin/bash /root/start.sh",

```

Comment:
>/start.sh [](start = 33, length = 9)

what is start.sh? I didn't see it in the docker file

Reply:
I put the start.sh file in the image "openpai/pai.example.kafka". And in the dockerfile, I just include the kafka environment, not the logic code(like start.sh, consumer.py, and producer.py). I mean, I think I just need to produce an environment for users in the dockerfile. If the users want to take the logic code, I'd like him to use the kafka.json instead. So, do I need to upload the logic code?